rust/migrate-wicked/tests/test.sh

```diff
@@ -51,6 +53,9 @@ while [[ $# -gt 0 ]]; do
     -v|--verbose)
       LOG_LEVEL=$((LOG_LEVEL + 1))
       ;;
+    -vv|--very-verbose)
+      set -x
+      ;;
```

Comment:
The intention of `-v` was, that you can actually do it like `-v -v -v` to get more verbosity.
So `-vv` would be:
```suggestion
    -vv|--very-verbose)
      LOG_LEVEL=$((LOG_LEVEL + 2))
      ;;
```

Maybe you do it like:
```
if [ $LOG_LEVEL -gt 2 ]; then 
  set -x 
fi
```

Or we call your option `--debug`

Reply:
or `man 1 getopt`

Reply:
Yeah the problem is that then it still executes the `log_verbose` functions, which is just noise if everything is printed out anyway. So I guess I'll change it to `--debug`